docs/resources/destination_adobe_analytics.md

```diff
@@ -0,0 +1,235 @@
+- `event_merch_event_to_adobe_event` (List of Object) You can map Rudder Events to Adobe Merchandise events. (see [below for nested schema](#nestedatt--config--event_merch_event_to_adobe_event))
+- `event_merch_properties` (List of String) Currency/Incremental properties to add to merchandise events at event level
+- `events_to_types` (List of Object) You can map your Rudder video events with types of Video Events (see [below for nested schema](#nestedatt--config--events_to_types))
+- `heartbeat_tracking_server_url` (String) Enter your Heartbeat Tracking Server URL
+- `hier_mapping` (List of Object) You can map Rudder Properties to Adobe Hierarchy properties (see [below for nested schema](#nestedatt--config--hier_mapping))
+- `list_delimiter` (List of Object) You can map your Rudder Property with Delimiters for list properties (see [below for nested schema](#nestedatt--config--list_delimiter))
+- `list_mapping` (List of Object) You can map Rudder Properties to Adobe list properties (see [below for nested schema](#nestedatt--config--list_mapping))
+- `marketing_cloud_org_id` (String) Enter your Marketing Cloud Organization Id.
+- `mobile_event_mapping` (List of Object) You can map Rudder Mobile events. (see [below for nested schema](#nestedatt--config--mobile_event_mapping))
+- `no_fallback_visitor_id` (Boolean) Check to enable no Fallbacks for Visitor ID
+- `onetrust_cookie_categories` (Block List, Max: 1) Specify the OneTrust category name for mapping the OneTrust consent settings to RudderStack's consent purposes. (see [below for nested schema](#nestedblock--config--onetrust_cookie_categories))
+- `page_name_fallback_tostring` (Boolean) Check to allow Page Name Fallback to Screen
+- `prefer_visitor_id` (Boolean) Check to prefer Visitor Id
+- `product_identifier` (String) Enter your Product Identifier
+- `product_merch_evars_map` (List of Object) You can map Rudder Properties to eVars at product level (see [below for nested schema](#nestedatt--config--product_merch_evars_map))
+- `product_merch_event_to_adobe_event` (List of Object) You can map Rudder Events to Adobe Merchandise events (see [below for nested schema](#nestedatt--config--product_merch_event_to_adobe_event))
+- `product_merch_properties` (List of String) Currency/Incremental properties to add to merchandise events at product level
+- `props_delimiter` (List of Object) You can map your Rudder Property with Delimiters for Adobe Custom properties (see [below for nested schema](#nestedatt--config--props_delimiter))
+- `proxy_heartbeat_url` (String) Enter your Adobe Analytics Hearbeat SDK URL
+- `proxy_normal_url` (String) Enter your Adobe Analytics Javascript SDK URL
+- `rudder_events_to_adobe_events` (List of Object) You can map Rudder Events to Adobe Custom Events. (see [below for nested schema](#nestedatt--config--rudder_events_to_adobe_events))
```

Comment:
```suggestion
- `rudder_events_to_adobe_events` (List of Object) You can map Rudder events to Adobe custom events. (see [below for nested schema](#nestedatt--config--rudder_events_to_adobe_events))
```

Reply:
@yashasvibajpai One thing to notice here is that PRs should first be made to the `develop` branch. Merging to `main` is the final action that should take place in order to actually publish the new version

Reply:
@fxenik Got it, shall I rebase the branch to develop?